### DISCUSSION TITLE
SSH connections panel

### DISCUSSION THREAD

**Discussion by liaskivskiy** [Eugeny/tabby#3230] [General]
Is it possible to add panel with list of ssh connections that will be always available and pinned to left or right side of the window?
In my opinion its will be more easy and quick to  open new ssh connection. 
Such feature is present  in MobaXterm as side tab with sessions:
![image](https://user-images.githubusercontent.com/52051594/103084660-3642aa80-45e8-11eb-94e9-468353ed33f1.png)

This option could be switchable on application or appearance setting if user  want to turn on/of ssh connections panel.

**Comment by ouyangxudu** [comment]
yes, I also think this is a good idea to find the connect to a new connection.
